meta/engineering_principles.md

```diff
@@ -0,0 +1,159 @@
+clearly, publicly and be empathetic to the priorities of others.
+
+- Examples:
+
+  - Grape
+  - CocoaPods
+  - The Artsy Omakase
+
+### Incremental Revolution
+
+Introduce new technologies slowly and incrementally. Avoid re-writes. Build tools to allow hybrids of different
+types of technology when possible. Sometimes you need to make a big leap, but aim to approach them incrementally.
+
+Explore bleeding-edge technologies on projects with an end-date and can become safely classed "done". These can be
+used to inform decisions on long-running projects. Run spike projects when trying to settle between technology
+trade-offs.
+
+- Examples
+
+  - Emission / Reaction
+  - Stitch
+  - Metaphysics
+
+- Further Reading
+
+  - Building a Pod Library http://artsy.github.io/blog/2018/04/17/making-a-components-pod/
+  - http://artsy.github.io/blog/2017/09/05/Modernizing-Force/
+
+### Minimal Viable Process
+
+As a team grows, so does the need for processes. Find ways to get the value of process without additional work.
+Build automated tools to help others. Use systems like RFCs to broaden the availability of information on process
+changes.
+
+For meetings over 2-3 people, provide agendas and make as many people optional as possible. We've seen great
+success on automation via documentation for recurring meetings.
+
+- Examples
+
+  - Danger
+  - Peril
+  - RFC Process
+  - artsy/meta
+
+- Further Reading
+
+  - http://artsy.github.io/blog/2018/05/25/support-process/
+  - http://artsy.github.io/blog/2017/09/04/Introducing-Peril/
+  - http://artsy.github.io/blog/2018/02/02/artsy-apogee/
+
+### Leverage Your Impact
+
+We built an outsized reputation for our size as an engineering team via the other principles. This gives the
+ability to move local impact to a larger communal impact.
+
+Consider ways in which you can expand the scope of impact for your time:
+
+- Avoid private messaging when you could communicate in a public channel
+- Team presentations could be worked for public consumption
+- Internal notes could be turned into blog posts
+- Documentation on teams and processes could be open for others to reference and learn from
+- A regular meetup could be converted into a franchisable system for others to replicate
+
+The small improvements eventually add up so that you can do industry level work as a part of building Artsy.
+
+- Examples
+
+  - Artsy x React Native
+  - JavaScriptures
+  - Learn Swift at Artsy
+  - Artsy Blog
+  - artsy/meta
+  - Peer Labs
+  - http://artsy.github.io/blog/2015/04/03/artsy-engineering-compensation-framework/
+
+### De-silo Engineers
+
+Provide people the ability to work in many spaces and not feel blocked by another team. Common languages, common
+terminology and common idioms. There's definitely space for experimentation, but within a shared framework. The aim
+is not to stifle creativity, but increase individual scope.
+
+- Examples
```

Comment:
We could mention the fact that we intentionally try to allow teams/people to not silo from an organizational perspective by allowing ourselves to re-staff/move periodically.